packages/next/src/index.ts

```diff
@@ -12,7 +12,7 @@ export default class LogtoClient {
   private storage?: NextStorage;
   constructor(private readonly config: LogtoNextConfig) {}
 
-  handleSignIn = (redirectUri = `${this.config.baseUrl}/api/sign-in`): NextApiHandler =>
+  handleSignIn = (redirectUri = `${this.config.baseUrl}/api/sign-in-callback`): NextApiHandler =>
```

Comment:
How about rename it to `handleSignInCallback`, in order to align with other SDKs?

Reply:
we have both `handleSignIn` and `handleSignInCallback`.

Reply:
Oh, OK